e2e_samples/parking_sensors/README.md

```diff
@@ -209,15 +212,37 @@ More resources:
 
 ## How to use the sample
 
-### Prerequisites
+This sample has certain prerequisites for deployment, which will be detailed below. Please note that using the development container is optional. However, we recommend using the development container because it streamlines the setup process, ensures consistency across environments, and simplifies dependency management.
 
-1. [Github account](https://github.com/)
+### Setup
+
+Following the setup prerequisites, permissions, and deployment environment options.
+
+#### Prerequisites
+
+1. [Github account](https://github.com/) : If you do not have already, create your GitHub Account.
 2. [Azure Account](https://azure.microsoft.com/en-us/free/)
    - *Permissions needed*: ability to create and deploy to an azure [resource group](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/overview), a [service principal](https://docs.microsoft.com/en-us/azure/active-directory/develop/app-objects-and-service-principals), and grant the [collaborator role](https://docs.microsoft.com/en-us/azure/role-based-access-control/overview) to the service principal over the resource group.
-3. [Azure DevOps Project](https://azure.microsoft.com/en-us/products/devops/)
+3. [Azure DevOps Project](https://azure.microsoft.com/en-us/products/devops/) : Follow the documentation to create a new project, or use an existing project you wish to deploy these resources to.
    - *Permissions needed*: ability to create [service connections](https://docs.microsoft.com/en-us/azure/devops/pipelines/library/service-endpoints?view=azure-devops&tabs=yaml), [pipelines](https://docs.microsoft.com/en-us/azure/devops/pipelines/get-started/pipelines-get-started?view=azure-devops&tabs=yaml) and [variable groups](https://docs.microsoft.com/en-us/azure/devops/pipelines/library/variable-groups?view=azure-devops&tabs=yaml).
 
-#### Software pre-requisites if you don't use dev container<!-- omit in toc -->
+#### Deployment Environment
```

Comment:
as I mentioned, I think just using `Deployment Options` for the title would be simpler